addons/l10n_es/data/account_tax_data.xml

```diff
@@ -1107,7 +1107,41 @@
     <record id="account_tax_template_s_iva_ns" model="account.tax.template">
         <field name="description">S_IVA_NS</field>
         <field name="type_tax_use">sale</field>
-        <field name="name">No sujeto Repercutido</field>
+        <field name="name">No sujeto Repercutido (Servicios)</field>
+        <field name="chart_template_id" ref="l10n_es.account_chart_template_common"/>
+        <field name="amount" eval="0"/>
+        <field name="amount_type">percent</field>
+        <field name="tax_group_id" ref="tax_group_iva_0"/>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
+            (0,0, {
+                'factor_percent': 100,
+                'repartition_type': 'base',
+                'tag_ids': [ref('mod_303_61')],
+            }),
+
+            (0,0, {
```

Comment:
There is no tax in this one. Is it needed to put this line?

Reply:
Yes, this line is still needed.

Reply:
Thanks for confirming.